docs/en/reference/architecture.rst

```diff
@@ -26,26 +26,26 @@ Doctrine ORM Packages
 
 Doctrine ORM is divided into three main packages.
 
--  Common
--  DBAL (includes Common)
--  ORM (includes DBAL+Common)
+-  Persistence
+-  DBAL
+-  ORM (includes DBAL+Persistence)
 
 This manual mainly covers the ORM package, sometimes touching parts
-of the underlying DBAL and Common packages. The Doctrine code base
+of the underlying DBAL and Persistence packages. The Doctrine code base
 is split in to these packages for a few reasons and they are to...
 
 
 -  ...make things more maintainable and decoupled
--  ...allow you to use the code in Doctrine Common without the ORM
+-  ...allow you to use the code in Doctrine Persistence without the ORM
    or DBAL
 -  ...allow you to use the DBAL without the ORM
 
-The Common Package
+The Persistence Package
 ~~~~~~~~~~~~~~~~~~
 
-The Common package contains highly reusable components that have no
+The Persistence package contains highly reusable components that have no
```

Comment:
This sentence probably refers more to doctrine/collections than doctrine/persistence. Maybe it should be mentioned as well.